### PR TITLE
doc: add note to contributor docs about warranted PR's

### PR DESCRIPTION
**Pull request by kallewoof**

### CODE REVIEW

**Comment by laanwj** [comment]
ACK

**Comment by fanquake** [comment]
Thanks, utACK c72aa1f

**Comment by practicalswift** [comment]
ACK

**Comment by promag** [comment]
ACK 3431e1a.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ At this stage one should expect comments and review from other contributors. You
 can add more commits to your pull request by committing them locally and pushing
 to your fork until you have satisfied all feedback.
 
+Note: Code review is a burdensome but important part of the developer process, and as such, certain types of pull requests are rejected. In general, if the **improvements** do not warrant the **review effort** required, the PR has a high chance of being rejected. It is up to the PR author to convince the reviewers that the changes warrant the review effort, and if reviewers are "Concept NAK'ing" the PR, the author may need to present arguments and/or do research backing their suggested changes.
```
Comment (promag) [inline CONTRIBUTING.md]: nit, ... the development process, ...

Comment (kallewoof) [reply CONTRIBUTING.md]: Thanks, fixed.

**Comment by practicalswift** [comment]
ACK db213aa47f6c9a2f3945ddd14f5912f1e8b9c9fc

**Comment by jonasschnelli** [comment]
ACK db213aa47f6c9a2f3945ddd14f5912f1e8b9c9fc

**Pull request merged**